.circleci/config.yml

```diff
@@ -552,11 +552,14 @@ jobs:
           key: v6-gradle-wrapper-{{ .Branch }}-{{ checksum "~/workbench/api/gradle/wrapper/gradle-wrapper.properties" }}
       - restore_cache:
           name: "Restore api gradle cache"
-          key: v6-gradle-cache-{{ .Branch }}-{{ checksum "~/workbench/api/build.gradle" }}
+          keys:
+            - v6-gradle-cache-{{ .Branch }}-{{ checksum "~/workbench/api/build.gradle" }}
+            - v6-gradle-cache-
       - restore_cache:
           name: "Restore e2e yarn cache"
           keys:
             - v5-e2e-yarn-{{ .Branch }}-{{ arch }}-{{ checksum "~/workbench/e2e/yarn.lock" }}
+            - v5-e2e-yarn-
```

Comment:
When no cache is found, load the most recent valid cache by setting a fallback key with the `v5-e2e-yarn-` prefix. Example, CircleCI will load the `v5-e2e-yarn-main-` (the main branch) cache.

Same is done for the gradle cache.

<img width="1297" alt="Screen Shot 2021-12-22 at 7 54 40 PM" src="https://user-images.githubusercontent.com/35533885/147171577-c9046226-2ce4-428a-9935-224d4ca85e25.png">

https://circleci.com/blog/config-best-practices-dependency-caching/

Reply:
Thanks! Do we also need a default gradle-wrapper cache?

Reply:
yes, for the gradle task that generates the access token.

Reply:
I'm referring to line 552 here.  My question is: should this PR have 3 changes instead of 2

Reply:
I see, good catch. It would have been nice to keep the behavior same for all three cache. I made a PR [6098](https://github.com/all-of-us/workbench/pull/6098).